daprdocs/content/en/reference/cli/dapr-run.md

```diff
@@ -29,6 +29,7 @@ dapr run [flags] [command]
 | `--app-protocol`, `-P`         |                      | `http`                                                                             | The protocol (gRPC or HTTP) Dapr uses to talk to the application. Valid values are: `http` or `grpc` |
 | `--app-ssl`                    |                      | `false`                                                                            | Enable https when Dapr invokes the application                                                       |
 | `--components-path`, `-d`      |                      | `Linux & Mac: $HOME/.dapr/components`, `Windows: %USERPROFILE%\.dapr\components`   | The path for components directory                                                                    |
+| `--resources-path`     |                      |   | The path for resources directory                                                                    |
```

Comment:
This needs a local directory where you can put your subscriptions and resiliency policies like components. For example `Linux & Mac: $HOME/.dapr/resources`

Reply:
The docs need more updates than just the CLI. This is an important change and needs to be covered in other docs. We also need to update
1) This sentence "Resiliency specs are saved in the same location as components specs and are applied when the Dapr sidecar starts." here https://docs.dapr.io/operations/resiliency/resiliency-overview/
2) The declarative subscriptions topics needs to be updated https://docs.dapr.io/developing-applications/building-blocks/pubsub/subscription-methods/
3) I suggest there is a new Concepts topic created here https://docs.dapr.io/concepts/ that covers what are resources and how these are used. In other words what resources are and when/how these are used. 
4) Since this is also being added to daprd.exe, this document needs to be updated https://docs.dapr.io/reference/arguments-annotations-overview/